### ISSUE TITLE
Unexpected Error message when saving an alert with the name of an alert that already exists

### ISSUE THREAD

**Issue by rbeckman-nextgen** [nextgenhealthcare/connect#2834]
Create an alert and save.  Now create another alert with the same name and save.  A dialog displays indicating an unexpected error occurred with an HTTP 500 and an alert with that name already exists:

com.mirth.connect.client.core.ClientException: method failed: HTTP/1.1 500 An alert with that name aleady exists.
com.mirth.connect.client.core.ServerConnection.executePostMethod(ServerConnection.java:130)
com.mirth.connect.client.core.Client.updateAlert(Client.java:423)
com.mirth.connect.client.ui.alert.DefaultAlertEditPanel.saveAlert(DefaultAlertEditPanel.java:182)
com.mirth.connect.client.ui.Frame$45.doInBackground(Frame.java:4025)
com.mirth.connect.client.ui.Frame$45.doInBackground(Frame.java:4022)
javax.swing.SwingWorker$1.call(Unknown Source)
java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
java.util.concurrent.FutureTask.run(Unknown Source)
javax.swing.SwingWorker.run(Unknown Source)
java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
java.lang.Thread.run(Unknown Source)

Obviously, an error message **should** appear here, but one a little more elegant than an unexpected exception message.

Imported Issue. Original Details:
Jira Issue Key: MIRTH-2908
Reporter: ericb
Created: 2013-08-29T20:01:54.000-0700

**Comment by rbeckman-nextgen** [comment]
Note that it is possible when saving a channel to get the same long message with exception, although most of the time you would only see a short message with no exception. I've made alerts show the same short message.

Also, when saving an alert with a name that already exists, an error message will still be logged. I've described why this is the case and marked it to be fixed in the future in MIRTH-3035

Imported Comment. Original Details:
Author: wayneh
Created: 2013-10-29T16:51:38.000-0700
